app/(articles)/[displayStyle]/[page]/page.tsx

```diff
@@ -41,9 +42,9 @@ export const generateStaticParams = async () => {
     const listPages = getPagesArray(listLastPage, MAX_PAGES);
 
     const paths = [
-      ...gridPages.map((page) => ({ displayStyle: 'grid', page })),
-      ...listPages.map((page) => ({ displayStyle: 'list', page })),
-    ];
+      ...gridPages.map((page) => ({ displayStyle: 'grid' as const, page })),
+      ...listPages.map((page) => ({ displayStyle: 'list' as const, page })),
+    ] satisfies readonly HomePageProps['params'][];
 
     return paths;
   } finally {
```

Comment:
`try…finally` nie jest potrzebne

Reply:
Więc `await closeConnection();` powinno znaleźć się przed returnem czy nie jest tutaj potrzebne?

Reply:
W ogóle nie jest potrzebne; Kiedyś Prisma miała problem z zamykaniem połączeń w Next.js. Od dawna jest to naprawione.